Formula/mpage.rb

```diff
@@ -0,0 +1,21 @@
+class Mpage < Formula
+  desc "Mpage, many to one page printing utility"
+  homepage "http://www.mesa.nl/pub/mpage/"
+  url "http://www.mesa.nl/pub/mpage/mpage-$version.tgz"
+  version "2.5.6"
```

Comment:
I misinterpreted what `brew audit` wanted me to do, and I'm pretty sure I installed the formula with this syntax. In any case I've put the version in the URL and removed the call to url.